README.md

```diff
@@ -4,17 +4,21 @@ Image Clipper
 This is a fork of the [imageclipper](https://code.google.com/p/imageclipper/)
 program initially written by Naotoshi Seo.
 
-This fork was made initially for myself to be able to compile the program on OSX.
-Since I used [CMake](http://www.cmake.org/) it was easy to make one build that
-works on all platforms.
+This fork was made initially for myself to improve its capabilities. 
+
 
 From the original description:
 
-> It is often required to crop images manually fast for computer vision researchers
-> to gather training and testing image sets, e.g., cropping faces in images.
+> It is often required to crop images manually fast for computer vision researchers to gather training and testing image sets, e.g., cropping faces in images.
```

Comment:
Please remove all changes to the README for this Pull Request, they are not relevant.